TOOLS/mpv-osd-symbols.sfdir/README.md

```diff
@@ -0,0 +1,16 @@
+Steps to add new icons:
+
+- Install `fontforge`
+- Install the last freely licensed version of symbola (`ttf-symbola-free` in the
+  AUR)
+- `fontforge /usr/share/fonts/TTF/Symbola.ttf TOOLS/mpv-osd-symbols.sfdir`
```

Comment:
Load a font and the project directory, for example: `fontforge Symbola.ttf TOOLS/mpv-osd-symbols.sfdir`